doajtest/functional/bulk_api.py

```diff
@@ -37,7 +37,7 @@
 # you'll change all of them
 dataset = []
 for i in range(0, 10):
-    dataset.append(deepcopy(ArticleFixtureFactory.make_incoming_api_article()))
+    dataset.append(deepcopy(DoajXmlArticleFixtureFactory.make_incoming_api_article()))
```

Comment:
So, related to my earlier comment, this method `make_incoming_api_article` returns JSON that's formatted according to the API, so I don't think it should be on `DoajXmlArticleFixtureFactory`.